assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -44,16 +44,20 @@ export type PackageRateRenderOption = (
 	option: CartShippingPackageShippingRate
 ) => PackageRateOption;
 
+// A flag can be trinary if true, false, and undefined are all valid options.
+// In our case, we use this for collapsible and showItems, having a boolean will force that
+// option, having undefined will let the component decide the logic based on other factors.
+export type TrinaryFlag = boolean | undefined;
 interface PackageProps {
 	/* PackageId can be a string, WooCommerce Subscriptions uses strings for example, but WooCommerce core uses numbers */
 	packageId: string | number;
 	renderOption: PackageRateRenderOption;
 	collapse?: boolean;
```

Comment:
This PR introduced a new TS error for https://github.com/woocommerce/woocommerce-blocks/blob/b68114987401bee56e3811bbe3b701507c30a8a0/assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx#L63-L72

I suggest applying the following change to prevent this TS error from appearing:
```suggestion
	collapse?: TrinaryFlag;
```